### PR TITLE
Animate alert view

### LINKED ISSUE #65: Add listview layout animations for alerts

### PR DESCRIPTION
**Pull request by styliii**
- resolves #65 

Just using RecyclerView default animator.  Couldn't get any of the custom libraries to work.  If you want to see it in action, comment out L113 in AlertsFragment.java.

### CODE REVIEW

File: WealthNOW/.idea/gradle.xml [hunk 1]
```diff
@@ -6,7 +6,6 @@
         <option name="distributionType" value="LOCAL" />
```
Comment (mikeng13) [inline WealthNOW/.idea/gradle.xml]: We should gitignore this file

File: WealthNOW/.idea/misc.xml [hunk 2]
```diff
@@ -3,7 +3,8 @@
   <component name="EntryPointsManager">
```
Comment (mikeng13) [inline WealthNOW/.idea/misc.xml]: I think Andrew might've added this to gitignore so oyu might have to merge in master for that.

File: WealthNOW/app/src/main/java/com/finnerds/wealthnow/adapters/AlertArrayAdapter.java [hunk 3]
```diff
@@ -1,129 +0,0 @@
-package com.finnerds.wealthnow.adapters;
```
Comment (mikeng13) [inline WealthNOW/app/src/main/java/com/finnerds/wealthnow/adapters/AlertArrayAdapter.java]: Delete this file?

File: WealthNOW/app/src/main/java/com/finnerds/wealthnow/fragments/HomeFragment.java [hunk 4]
```diff
@@ -248,7 +240,7 @@ private void setupBarChart(boolean shouldAnimate) {
 
         ArrayList<String> stockSymbols = new ArrayList<>();
         ArrayList<BarEntry> changeValues = new ArrayList<>();
-        for (int i = 0; i < Math.max(copyOfStocks.size(), 6); i++) {
+        for (int i = 0; i < Math.min(copyOfStocks.size(), 6); i++) {
```
Comment (mikeng13) [inline WealthNOW/app/src/main/java/com/finnerds/wealthnow/fragments/HomeFragment.java]: Thanks for this!

File: WealthNOW/app/src/main/java/com/finnerds/wealthnow/helpers/DividerItemDecoration.java [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+import android.content.res.TypedArray;
+import android.graphics.Canvas;
+import android.graphics.Rect;
+import android.graphics.drawable.Drawable;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+
+/*
+ * Copyright (C) 2014 The Android Open Source Project
```
Comment (mikeng13) [inline WealthNOW/app/src/main/java/com/finnerds/wealthnow/helpers/DividerItemDecoration.java]: Did we pull this from somewhere? Can we add the URL if so.

**Comment by mikeng13** [comment]
Looking good! I haven't tried it on my phone yet. Hoping to get to it later today.

File: WealthNOW/app/src/main/java/com/finnerds/wealthnow/adapters/AlertArrayAdapter.java (hunk 3 above)
Comment (styliii) [inline WealthNOW/app/src/main/java/com/finnerds/wealthnow/adapters/AlertArrayAdapter.java]: yes, the AlertRecyclerAdapter replaces it

**Pull request merged**